src/main/java/fact/utils/ElementwiseMultiplyDoubleArray.java

```diff
@@ -0,0 +1,46 @@
+package fact.utils;
+
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
```

Comment:
So what you telling me is, I should in this PR move `Elementwise` somewhere else so I can use it in its intended scope? Am I understanding you correctly? 

You said "this class" above .. and I was not sure which class you are talking about. 
Your comment strongly felt, like it was part of a discussion which I missed. 

So please make yourself clear here. Should I touch this other code within this PR? Is this necessary to make this PR get your approval, or not?